lib/fluent/agent.rb

```diff
@@ -158,35 +159,5 @@ def emit_error_event(tag, time, record, error)
 
     def handle_emits_error(tag, es, error)
     end
-
-    class NoMatchMatch
```

Comment:
I checked the code of `out_reemit`. It doesn't refer `Fluent::NoMatchMatch`.
(It just refer `Fluent::EngineClass::NoMatchMatch` in `V10Engine`).
Isn't it?